CommunityFridgeMapApi/functions/email_service/v1/app.py

```diff
@@ -0,0 +1,36 @@
+import logging
+import json
+
+from send_email import sendEmail
+
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
+recipient = "info@collectivefocus.site"
+
+
+class ContactUs:
+    @staticmethod
+    def lambda_handler(event: dict) -> dict:
```

Comment:
maybe a unit test would be good for this function?

Reply:
sure thanks, i'll add one